doc/api/build.md

```diff
@@ -70,3 +70,40 @@ GET /api/application/{application_id}/build/latest
   "download_link": "http://app-build.dev/app_dev.php/fr/application/1/build/1/download"
 }
 ```
+
+## Create a new build (WIP)
+
+### Request
+
+```
+PUT /api/application/{application_id}/build
+```
+```json
+{
+  "version": "1.5.2",
+  "comment": "This is a comment for this build.\nWith a line break.\nAnd another one.\n"
+}
+```
+
+### Response headers
+
+```
+Location: http://domain.com/application/3/build/1/file
```

Comment:
Je vais le changer de toute façon, c'est trop galère de changer le header Location, et ça n'a pas vraiment de sens. Dans le code, je retourne un json avec build id et url to upload